lib/src/components/NxPagination/__tests__/NxPagination.test.tsx

```diff
@@ -252,10 +252,9 @@ describe('NxPagination', function() {
   });
 
   describe('accessibility', function() {
-    it('includes the current and max page in the nav label and can be tabbed to for screen readers', function() {
+    it('includes an aria-label in the nav', function() {
       const wrapper = getShallowComponent({ pageCount: 3, currentPage: 0 });
-      expect(wrapper).toHaveProp('aria-label', 'pagination, page 1 of 3');
-      expect(wrapper).toHaveProp('tabIndex', 0);
+      expect(wrapper).toHaveProp('aria-label', 'pagination');
     });
```

Comment:
Needs tests that the caller can override the `aria-label`

Reply:
> Needs tests that the caller can override the `aria-label`

Added here: a435b0956919196337ce601892c2ab0a390495a1